docs/plugins/zero.rst

```diff
@@ -18,6 +20,9 @@ fields to nullify and the conditions for nullifying them:
   get the list of all available fields by running ``beet fields``. In
   addition, the ``images`` field allows you to remove any images
   embedded in the media file.
+* Set ``keep_fields`` respectively to list of fields that plugin should
+  preserve. That way ``zero`` cleans anything other than fields written in this
+  option.
```

Comment:
To make this a little clearer, I suggest:

    Set ``keep_fields`` to *invert* the logic of the plugin. Only these fields
    will be kept; other fields will be removed. Remember to set only
    ``fields`` or ``keep_fields``, not both!